ui/src/pages/Questions/Linked/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { FC, useState } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import { useParams, useSearchParams, Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
+
+import { usePageTags } from '@/hooks';
+import { questionLink, questionDetail } from '@/services';
+import * as Type from '@/common/interface';
+import {
+  QuestionList,
+  CustomSidebar,
+  HotQuestions,
+  FollowingTags,
+} from '@/components';
+import { userCenter, floppyNavigation } from '@/utils';
+import { QUESTION_ORDER_KEYS } from '@/components/QuestionList';
+import {
+  loggedUserInfoStore,
+  siteInfoStore,
+  loginSettingStore,
+} from '@/stores';
+
+const LinkedQuestions: FC = () => {
+  const { qid } = useParams<{ qid: string }>();
+  const { t } = useTranslation('translation', { keyPrefix: 'linked_question' });
+  const { t: t2 } = useTranslation('translation');
+  const { user: loggedUser } = loggedUserInfoStore((_) => _);
+  const [urlSearchParams] = useSearchParams();
+  const curPage = Number(urlSearchParams.get('page')) || 1;
+  const curOrder = (urlSearchParams.get('order') ||
+    QUESTION_ORDER_KEYS[0]) as Type.QuestionOrderBy;
+  const pageSize = 10;
+  const { siteInfo } = siteInfoStore();
+  const { data: listData, isLoading: listLoading } = questionLink({
+    question_id: qid || '',
+    page: curPage,
+    page_size: pageSize,
+  });
+  const { login: loginSetting } = loginSettingStore();
+  const [questionTitle, setQuestionTitle] = useState('');
+  questionDetail(qid || '')
+    .then((res) => {
+      setQuestionTitle(res.title);
+    })
+    .catch((err) => {
+      console.error('get question detail failed:', err);
+      setQuestionTitle(`#${qid}`);
+    });
+  usePageTags({
+    title: t('title'),
+  });
+
+  return (
+    <Row className="pt-4 mb-5">
+      <Col className="page-main flex-auto">
+        <h3>Linked Questions</h3>
+        <p>
```

Comment:
add class .mb-5, and use <div>  replace <p> tag

 <div className='mb-5'>
          Questions linked to/from&nbsp;
          <a href={`/questions/${qid}`}>{questionTitle}</a>
        </div>